.gitlab-ci.yml

```diff
@@ -85,6 +86,42 @@ build:make:
   tags:
     - docker
 
+smoketests:mac:
+  stage: smoketests
+  dependencies:
+    - build:make
+  before_script:
+    - wget https://dgsbl4vditpls.cloudfront.net/mender-demo-artifact.mender
+    - touch test.txt
+  script:
+    - ./mender-artifact-darwin
+    - ./mender-artifact-darwin --version
+    - ./mender-artifact-darwin validate mender-demo-artifact.mender
+    - ./mender-artifact-darwin write module-image -t test -o test.mender -T script -n test -f test.txt
```

Comment:
```suggestion
    - ./mender-artifact-darwin write module-image -t test -o test.mender -T script -n test -f test.txt
    - ./mender-artifact-darwin read test.mender
    - ./mender-artifact-darwin write rootfs-image -o test-rfs.mender -n test -f test.txt
    - ./mender-artifact-darwin read test-rfs.mender
```

Reply:
good point, should now be the case for both mac & linux 👌